locales/en.json

```diff
@@ -1,6 +1,6 @@
 {
   "settings": {
-    "offlineNodeSaveMessage": "Settings will be saved during the next wakeup of this battery device."
+    "offlineNodeSaveMessage": "Settings will be saved during the next wakeup of this battery device. This might take a while, activate 'learn mode' to wake the device manually."
```

Comment:
```suggestion
    "offlineNodeSaveMessage": "Settings will be saved during the next automatic wakeup of this battery device, this might take a while. Manually activate 'learn mode' on the device to update settings immediately."
```

Reply:
I like it!